server/helpers_test.go

```diff
@@ -36,3 +38,34 @@ type stubOCSPRespSource struct {
 func (s *stubOCSPRespSource) Fetch(*certchain.RawChain, func() time.Time) (*certchain.OCSPResponse, futureevent.Event, error) {
 	return s.data, futureevent.NeverOccurs(), nil
 }
+
+//------------------------
```

Comment:
All comments to the other `stubCache` apply here too.

Reply:
Done